README.md

```diff
@@ -4,22 +4,23 @@
-| [ICRC-49](./topics/icrc_49_call_canister.md)                                                                       | Call Canister<br>[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./topics/icrc_25_signer_interaction_standard.md)                 | [Philipp Litzenberger](https://github.com/plitzenberger)          | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
+| [ICRC-49](./topics/icrc_49_call_canister.md)                                                                      | Call Canister<br>[![Extension Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./topics/icrc_25_signer_interaction_standard.md)                 | [Philipp Litzenberger](https://github.com/plitzenberger)          | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Signer Standards Overview](https://github.com/dfinity/wg-identity-authentication/blob/main/topics/signer_standards_overview.md)           |
 | [ICRC-#]((https://github.com/dfinity/wg-identity-authentication/issues/26) )                                      | DID registry                                                                                                                                                      | [Quint Daenen](https://github.com/q-uint)                         | [![Status Badge](https://img.shields.io/badge/STATUS-ISSUE-blue.svg)](https://github.com/orgs/dfinity/projects/31)   | [w3c DID registries](https://www.w3.org/TR/did-spec-registries/)                                                                           |
-| [CAIP-2](https://github.com/icvc/icp-namespace/pull/1)<br>[CAIP-10](https://github.com/icvc/icp-namespace/pull/2) | CAIP Specifications                                                                                                                                               | [Quint Daenen](https://github.com/q-uint)                         | [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31) | [Issue 25](https://github.com/dfinity/wg-identity-authentication/issues/25), [CAIP Specifications](https://github.com/ChainAgnostic/CAIPs) |
+| [CAIP-2](https://github.com/icvc/icp-namespace/pull/1)<br>[CAIP-10](https://github.com/icvc/icp-namespace/pull/2) | CAIP Specifications  
```

Comment:
You should probably not remove this. At least not without talking to Quint first. 😉

Reply:
Fixed also double checked that nothing was removed and correctly updated the table column alignment